mmv1/products/iap/api.yaml

```
@@ -237,3 +237,125 @@ objects:
         description: |
           Human-friendly name given to the OAuth client.
         required: true
+  - !ruby/object:Api::Resource
+    name: 'Settings'
```

```suggestion
    name: 'WebBackendServiceSettings'
```

Rather than just `Settings` I think we need to scope this on the parent, as above. See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_web_backend_service_iam for an example of a prior resource that does so, the IAM implementation of IAP w/ this target.

If it was multiparent, `Settings` would be appropriate, but given it's single-parent we'll want to scope it.